src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {

```

Comment:
Don't use verbs for class/interface names. Consider renaming `AreaCalculator` to something like `AreaCalculable` to better align with Java interface naming conventions.